website/content/docs/operations/session-recordings/validate-session-recordings.mdx

```diff
@@ -1,8 +1,8 @@
 ---
 layout: docs
-page_title: Validate the integrity recorded sessions
-description: |-
-  How to validate the integrity of Boundary's recorded sessions
+page_title: Validate recorded sessions' integrity
```

Comment:
does the plural add any value here? 

```suggestion
page_title: Validate recorded session integrity
```

Reply:
Nah, I don't think it does and people frequently struggle with this plural possessive form. Good catch!